docs/contracts.rst

```diff
@@ -723,22 +723,17 @@ the contract can only see the last 256 block hashes.
 
 Up to three parameters can
 receive the attribute ``indexed`` which will cause the respective arguments
-to be searched for: It is possible to filter for specific values of
-indexed arguments in the user interface.
-
-If arrays (including ``string`` and ``bytes``) are used as indexed arguments, the
-Keccak-256 hash of it is stored as topic instead.
-
-The hash of the signature of the event is one of the topics except if you
+to be stored in a special data structure as so-called "topics", which allows them to be searched for,
```

Comment:
Should mention the size of each topic (e.g. 256bit?). Perhaps a link to a description of these could be useful, if there's anything outside the YP :)

Reply:
I don't think the size is relevant. We also do not explain how "topics" are stored, so why should we care about the size? I will explain below why non-word values are hashed.